controller/common/node_info.py

```diff
@@ -52,5 +55,16 @@ def is_array_iscsi_iqns_match(self, host_iqns):
         host_iqns_lower = [iqn.lower() for iqn in host_iqns]
         return self._iscsi_iqn_lowercase in host_iqns_lower
 
+    def is_array_nvme_nqn_match(self, host_nqns):
+        """
+        Args:
+           host_nqns: storage host nqns list
+
+        Returns:
+           Is current host nqns matches
+        """
+        host_nqns_lower = [nqn.lower() for nqn in host_nqns]
+        return self._nvme_nqn_lowercase in host_nqns_lower
+
     def __str__(self):
-        return "iscsi_iqn: " + self.iscsi_iqn + ", fc_wwns: " + ",".join(self.fc_wwns)
+        return "iscsi_iqn: " + self.iscsi_iqn + ", fc_wwns: " + ",".join(self.fc_wwns) + "nvme_nqn: " + self.nvme_nqn
```

Comment:
consider choosing a consistent order for the connectivity types wherever applicable

Reply:
I mean [throughout](https://github.com/IBM/ibm-block-csi-driver/blob/574a573978764d487dd86c8e1581b04f32e86cc6/controller/common/node_info.py#L22) [the](https://github.com/IBM/ibm-block-csi-driver/blob/574a573978764d487dd86c8e1581b04f32e86cc6/controller/array_action/array_mediator_svc.py#L637) [PR](https://github.com/IBM/ibm-block-csi-driver/blob/574a573978764d487dd86c8e1581b04f32e86cc6/controller/array_action/array_mediator_svc.py#L1106)...

Reply:
then let's choose the order with minimum effort.
I don't think it matters what order, as long as it is consistent

Reply:
done